pages/items/[slug]/manage/details.tsx

```diff
@@ -44,8 +44,6 @@ const query = graphql`
       ...SchemaInstanceFormFragment
       ...SchemaSelectorDataFragment
     }
-    schemaVersions {
-      ...SchemaSelectorOptionsFragment
-    }
+    ...SchemaInstanceFormSchemaOptionsFragment
```

Comment:
Nice! less to remember on parent queries. I can add this to my PR and get it built early Monday, before demo.

Reply:
Sweet! Glad it helps! I only made this change for the one detail view — I imagine it will have to be applied to drawers, collection, etc.